docker-compose.yml

```diff
@@ -84,26 +83,11 @@ services:
       AGENT_NAME: ${CAM_AGENT_NAME}
       AGENT_PASSWORD: ${CAM_AGENT_PASSWORD}
       CAM_GW_URL: "https://${ANMS_GW_FQDN}:443/"
-      PROXY_URLS: "http://nginx/"
+      PROXY_URLS: "core,http://anms-core:5555 grafana,http://grafana:3000 grafana/api/live,http://grafana:3000 renderer,http://grafana-image-renderer:8081 adminer,http://adminer:8080 nm,http://ion-manager:8089 http://anms-ui:9030"
       NOT_ENFORCED_URLS: "http://nginx/favicon.ico"
     volumes:
       - ammos-tls:/ammos/etc/pki/tls:ro
 
-  nginx:
-    hostname: nginx
-    image: ${DOCKER_IMAGE_PREFIX}anms-nginx:${DOCKER_IMAGE_TAG}
-    build:
-      context: nginx
-      dockerfile: Dockerfile
-    restart: unless-stopped
-    security_opt:
-      - "label=type:nginx.process"
```

Comment:
This should also remove the associated nginx config in `puppet/modules/anms/files/selinux/nginx.cil` and reference to it in `puppet/modules/anms/manifests/init.pp`.